docker-setups/angular-docker.md

```diff
@@ -310,4 +312,17 @@ Setting up a Docker environment for your Angular application with Nginx offers s
 
 Consider enhancing your Docker setup by adding features such as environment-specific configurations, health checks, and Docker Swarm/Kubernetes integration for production deployments.
 
+[Back to top](#table-of-contents)
+
+### Remember to:
+
+- Maintain consistent naming conventions for services, container_name, image, and volume.
+- Add COMPOSE_PROJECT_NAME in .env.example and .env files to avoid conflicts with other projects.
+```plaintext
+COMPOSE_PROJECT_NAME=project-name
+```
+- Use docker-compose for setting up the containers instead of manual building processes.
+- Map the ports properly like this "127.0.0.1:5000:5000" to avoid security risks.
```

Comment:
Mention that we should add `127.0.0.1` to map and expose port only locally.

Reply:
Updated the comment as per your instruction.